### PR TITLE
Updated Plugin Development Document for changes to HelloWorldServlet for Tenant Parameter

### PR DESCRIPTION
**Pull request by learnjavawithreshma**

### CODE REVIEW

File: userguide/tutorials/plugin_development.adoc [hunk 1]
```diff
@@ -134,6 +134,40 @@ listening on:
...
+        	logger.info("tenant is available");
+        	Tenant t1 = tenant.get();
+        	logger.info("tenant id:"+t1.getId());
+        }
+        else {
+        	logger.info("tenant is not available");
+        }
+    }
+
+* This method provides the code for the \http://localhost:8080/plugins/hello-world-plugin end point.
```
Comment (pierre) [inline userguide/tutorials/plugin_development.adoc:156]: Nit: `endpoint` (one word).

**Review by pierre — APPROVED** [review]
Just one small typo.

**Comment by pierre** [comment]
Merging to update the mailing-list post.

File: userguide/tutorials/plugin_development.adoc (hunk 1 above)
Comment (learnjavawithreshma) [reply userguide/tutorials/plugin_development.adoc:156]: Fixed

**Pull request merged**